src/components/Initiatives/InitiativeDetailsPage/InitiativeDetailsPage.tsx

```diff
@@ -55,48 +63,140 @@ export const InitiativeDetailsPage = () => {
+        return (
+          <InitiativeFailingTab
+            actionItems={filteredActionItems}
+            entitiesByTag={entitiesByTag}
+            numRules={initiative.rules.length}
+            scorecardId={initiative.scorecard.id}
           />
-          <InitiativeTableCard
+        );
+      case InitiativeDetailsTab.Passing:
+        return (
+          <InitiativePassingTab
             actionItems={actionItems}
             componentRefs={filteredComponentRefs}
             entitiesByTag={entitiesByTag}
-            numRules={initiative.emphasizedRules.length}
+            numRules={initiative.rules.length}
+            scorecardId={initiative.scorecard.id}
           />
+        );
+      case InitiativeDetailsTab.Levels:
+        return (
+          <InitiativeLevelsTab
+            levels={initiative.levels}
+            rules={initiative.rules}
+          />
+        );
+      case InitiativeDetailsTab.Rules:
+        return <InitiativeRulesTab rules={initiative.rules} />;
+    }
+  };
+
+  return (
+    <Content>
+      <Grid container direction="row" spacing={1}>
+        <Grid item lg={5} xs={12}>
+          <InitiativeMetadataCard initiative={initiative} />
+        </Grid>
+        <Grid item lg={7} xs={12}>
+          <Box display="flex" flexDirection="column">
+            <InitiativeStatsCard
+              scores={initiative?.scores ?? []}
+              actionItems={filteredActionItems}
+              filter={filter}
+            />
+          </Box>
         </Grid>
       </Grid>
+
+      <Box display="flex" justifyContent="space-between" alignItems="center">
+        <Tabs
+          value={activeTab}
+          indicatorColor="primary"
+          textColor="primary"
+          onChange={handleTabChange}
+        >
+          <Tab
+            label={InitiativeDetailsTab.Failing}
+            value={InitiativeDetailsTab.Failing}
+          />
+          <Tab
+            label={InitiativeDetailsTab.Passing}
+            value={InitiativeDetailsTab.Passing}
+          />
+          {!isEmpty(initiative?.levels) ? (
+            <Tab
+              label={InitiativeDetailsTab.Levels}
+              value={InitiativeDetailsTab.Levels}
+            />
+          ) : (
+            <Tab
+              label={InitiativeDetailsTab.Rules}
+              value={InitiativeDetailsTab.Rules}
+            />
+          )}
+        </Tabs>
+        {activeTab === InitiativeDetailsTab.Failing && (
+          <Button
+            onClick={() => setIsFilterDialogOpen(true)}
+            variant="outlined"
+            aria-label="Filter"
+          >
+            filter
+          </Button>
+        )}
+      </Box>
+      <Box my={1}>{renderTab(activeTab)}</Box>
+      <InitiativeFilterDialog
+        isOpen={isFilterDialogOpen}
+        actionItems={actionItems}
+        initiative={initiative}
+        handleClose={handleFilterDialogClose}
+        setFilter={(filter: any) => setFilter(() => filter)}
```

Comment:
[Non-blocking] Is there a type we can use that's more descriptive than `any` here?